test/functional/toolchains/cmake/test_transitive_build_scripts.py

```diff
@@ -85,6 +86,40 @@ def build(self):
     assert "MYFUNCTION CMAKE A: Hello world A!!!" in c.out
 
 
+def test_transitive_build_scripts_error():
```

Comment:
Is this a functional test? Looks like an integration one (maybe add `light=True`)?

Reply:
Yes, this one is more an integration test, but it was developed here because of being very close to the topic of the other tests in this file, while I couldn't find a better location in integration that wouldn't be a new test file just for this test.